### DISCUSSION TITLE
Tehnicka specifikacija

### DISCUSSION THREAD

**Discussion by cugurm** [matf-pp/2021_MoneyCare#3] [General]
Na koje se tacno statisticke proracune misli?

**Comment by bogdanis7** [comment]
Za sada nam je plan da za svakog korisnika u bazi cuvamo podatke o njegovim primanjima i potrosnji novca po kategorijama (hrana, odeca, racuni...) i u skladu sa tim racunamo prosecnu potrosnju po odredjenim vremenskim periodima i preporucujemo korisniku racionalnu potrosnju novca. Dakle, svakako ce se na osnovu uzorka iz baze racunati ocekivanje i detektovati autlajeri i slicno (trenutno slusamo kurs statistike, pa cemo u skladu sa tim da vrsimo izmene), a ubacicemo i histograme, pie, barplotove...
Da li Vam ovo deluje u redu i da li imate neke preporuke?

**Reply by cugurm** [reply]
U redu. 

Potrudite se samo da deo projekta koji se odnosi na statistiku i ML ne bude fokus projekta, jer prema pravilima:

Nije dozvoljeno web programiranje niti samo upotreba baza podataka. Takođe, teme iz oblasti mašinskog učenja nisu prihvatljive.

Dakle fokus na GO, samu app, etc.
